src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.scss

```diff
@@ -52,8 +52,8 @@
     }
 
     .bcs-activity-feed-active-state {
-        padding: 9px 20px;
-        width: 100%;
+        padding: 9px 25px;
+        width: 340px;
```

Comment:
Why does this need to be hard coded to a px value rather than 100% like it was before?